ui/package.json

```diff
@@ -10,17 +19,24 @@
     "@types/node": "^12.0.0",
     "@types/react": "^16.9.53",
     "@types/react-dom": "^16.9.8",
+    "antd": "^4.9.4",
```

Comment:
a to do przegadania. Bo ja myślałam nad https://material-ui.com/ to musimy wspólnie ustalić w co wolimy iść

Reply:
Agree, that's why I decided to create a PR earlier, so we can talk about details :)

Can you create a task for that so we remember to talk about it?

Reply:
https://trello.com/c/0qRbMKad/11-wybranie-ui-design